gef.py

```diff
@@ -10647,6 +10681,21 @@ def invoke(self, args, from_tty):
         gdb.execute("gef help")
         return
 
+    def add_context_pane(self, pane_name, display_pane_function, pane_title_function):
+        """Add a new context pane to ContextCommand."""
+        for _, _, class_obj in self.loaded_commands:
```

Comment:
Weird we have to iterate over all loaded commands. Maybe in the future we should (also?) keep a map based on class name or command or something.

Reply:
This would be a nice idea to change, @Grazfather do you want it changed in this PR?

Reply:
Let's ask @hugsy. Another PR is fine, but maybe he doesn't want it at all.